lib/util/exec/index.ts

```diff
@@ -100,17 +100,29 @@ async function prepareRawExec(
   opts: ExecOptions = {}
 ): Promise<RawExecArguments> {
   const { docker } = opts;
-  const { customEnvVariables } = GlobalConfig.get();
+  const { customEnvVariables, cacheDir, binarySource } = GlobalConfig.get();
+
+  if (binarySource === 'docker' || binarySource === 'install') {
+    const buildPackCacheDir = upath.join(cacheDir, 'buildpack');
```

Comment:
@Chumper Do we need to create that folder in advance? Or does buildpack create required structure automatically? 🤔

Reply:
I think it will be created on the first download by curl, but it doesn't hurt to create it

Reply:
OK, don't like to add any fs code to exec, so would need to do it on init caches on worker code.